x-pack/plugins/snapshot_restore/common/lib/index.ts

```diff
@@ -6,10 +6,7 @@
  */
 
 export { flatten } from './flatten';
-export {
-  deserializeRestoreSettings,
```

Comment:
I couldn't find deserializeRestoreSettings being used anywhere so I removed it. Happy to put it back if anyone has a good reason for it to stay!